app/controllers/providers/partners/unexpected_employment_incomes_controller.rb

```diff
@@ -0,0 +1,37 @@
+module Providers
+  module Partners
+    class UnexpectedEmploymentIncomesController < ProviderBaseController
+      before_action :employments_and_payments
+
+      def show
+        @partner = partner
+        @form = LegalAidApplications::UnexpectedEmploymentIncomeForm.new(model: legal_aid_application)
+      end
+
+      def update
+        @partner = partner
```

Comment:
I think this could be simplified by just called the `partner` method which you've defined below, it seems like all that's happening here is that you're assigning the variable again, but that's already happening in the method (unless I've misunderstood!)
```suggestion
        partner
```